Library/Homebrew/test/fixtures/test.eps

```diff
@@ -0,0 +1,25 @@
+%!PS-Adobe-2.0 EPSF-2.0
+%%BoundingBox: 73 499 211 513
+%%HiResBoundingBox: 73.000000 499.500000 211.000000 512.500000
+%%EndComments
+% EPSF created by ps2eps 1.68
+%%BeginProlog
+save
+countdictstack
+mark
+newpath
+/showpage {} def
+/setpagedevice {pop} def
+%%EndProlog
+%%Page 1 1
+/Courier
+20 selectfont
+72 500 moveto
+(Hello World!) show
+showpage
+%%Trailer
+cleartomark
+countdictstack
+exch sub { end } repeat
+restore
+%%EOF
```

Comment:
I'm not too familiar to eps standard. This one is generated by `ps2eps` where ps file is from wikipedia.